parm/post_avblflds.xml

```diff
@@ -7709,6 +7709,469 @@
        <scale>6.0</scale>
     </param>
 
+    <param>
+       <post_avblfldidx>1019</post_avblfldidx>
```

Comment:
@hsinmulin-NOAA I can't find grib2 names of new fields you are proposing in the grib2 table https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_table4-2-0-13.shtml. Any grib2 tables I should look for?